torch/nested/_internal/sdpa.py

```diff
@@ -642,7 +656,8 @@ def _autocast(
     actual dtype conversions.
     """
     device_type = query.device.type
-    if not torch.is_autocast_enabled(device_type):
+    # meta device is not supported by autocast, so break early for it
+    if _is_computing_meta_flops(query) or not torch.is_autocast_enabled(device_type):
```

Comment:
Does this not fail usually anyway?

Reply:
`torch.is_autocast_enabled("meta")` throws an error